tests/foreman/api/test_computeresource_azurerm.py

```diff
@@ -149,6 +150,7 @@ def test_positive_check_available_networks(self, azurermclient, module_azurerm_c
         assert len(portal_nws) == len(cr_nws['results'])
 
 
+@run_in_one_thread
```

Comment:
yes, because in `run_in_one_thread` all tests run around with one instance and save time(during create) however in multi threads more instances could spin for same tests and increase the execution time.

Reply:
ACK, Needs to do the same for GCE.